app/models/concerns/externally_identifiable.rb

```diff
@@ -4,6 +4,19 @@ module ExternallyIdentifiable
   extend ActiveSupport::Concern
 
   module ClassMethods
+    def efind(use_id)
```

Comment:
When I first saw this method, I thought it was a typo. The name's a bit weird. Maybe `find_by_best_id` or `find_by_id_or_external_id` (verbose, I know)